camayoc/tests/qcs/cli/utils.py

```diff
@@ -48,25 +48,14 @@ def cred_show(options, output, exitstatus=0):
         network and \d+ respectively.
     :param exitstatus: Expected exit status code.
     """
-    expected_output = '{\r\n'
```

Comment:
I am normalizing all show helper to accept an regex pattern as the output argument